### ISSUE TITLE
fix Android 7.0 SecurityException

### ISSUE THREAD

**Issue by adrcotfas** [adrcotfas/Goodtime#32]
Stack trace:
```
java.lang.SecurityException: Not allowed to change Do Not Disturb state
	at android.os.Parcel.readException(Parcel.java:1683)
	at android.os.Parcel.readException(Parcel.java:1636)
	at android.media.IAudioService$Stub$Proxy.setRingerModeExternal(IAudioService.java:962)
	at android.media.AudioManager.setRingerMode(AudioManager.java:1022)
	at com.apps.adrcotfas.goodtime.MainActivity.A(Unknown Source)
	at com.apps.adrcotfas.goodtime.MainActivity.a(Unknown Source)
	at com.apps.adrcotfas.goodtime.MainActivity.c(Unknown Source)
	at com.apps.adrcotfas.goodtime.MainActivity.a(Unknown Source)
	at com.apps.adrcotfas.goodtime.e.onClick(Unknown Source)
	at android.view.View.performClick(View.java:5609)
	at android.view.View$PerformClick.run(View.java:22259)
	at android.os.Handler.handleCallback(Handler.java:751)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6077)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
```

**Comment by adrcotfas** [comment]
Fixed with 9aafa7d4106782532ea154a0d4323fdb9e4176f4

**Comment by adrcotfas** [comment]
Other improvements: 
- Avoid crash for users updating to Android N which have the disable sound option checked: 159a42b44f49a52ba8380e7ef26bda69a54242a4
- Clean-up: 8664153f58a7253095476654d557a10512b1721c
